scripts/setup-hooks.sh

```diff
@@ -3,8 +3,8 @@
 # Setup git pre-commit hooks for the project
 echo "Setting up pre-commit hooks..."
 
-# Check if .git directory exists
-if [ ! -d ".git" ]; then
+# Check if .git directory or file exists (supports both regular repos and worktrees)
+if [ ! -d ".git" ] && [ ! -f ".git" ]; then
```

Comment:
nit: could be just this:

```suggestion
if [ ! -e ".git" ]; then
```